packages/mainmenu-extension/style/index.css

```diff
@@ -3,4 +3,20 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
+.jp-LabShell[data-shell-mode='single-document'] #jp-top-panel #jp-MainLogo {
+  animation-duration: 1s;
+  animation-name: logo-slideup;
+  animation-fill-mode: forwards;
+}
+
+@keyframes logo-slideup {
```

Comment:
I'm not sure if this is needed, as the logo isn't really changing position from the user's point of view (in top left corner of view both before and after). At least that is my impression from the gif, not sure if there are other cases I'm missing?

Reply:
Removing in an upcoming commit